src/els_text_synchronization.erl

```diff
@@ -1,5 +1,7 @@
 -module(els_text_synchronization).
 
+-include("erlang_ls.hrl").
```

Comment:
Why was this added?

Reply:
Yes, I added it to fix compilation.

Aside: we should check that includes *are* actually needed.